sale_stock_picking_update_responsible/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1 @@
+* Iria Alonso <iria.alonso@solvos.es>
```

Comment:
For new `.md` format special chars `<` and `>` should be escaped. Otherwise `README.rst` will skip these characters when generated.